com.reprezen.swagedit.openapi3/META-INF/MANIFEST.MF

```diff
@@ -24,8 +24,7 @@ Require-Bundle: org.eclipse.ui,
  org.apache.log4j;bundle-version="1.2.15",
  com.reprezen.jsonoverlay;bundle-version="3.1.0",
  com.reprezen.kaizen.openapi-parser;bundle-version="3.0.1",
- com.norconex.commons.norconex-commons-lang;bundle-version="1.15.0",
- javax.mail-api;bundle-version="1.6.1"
+ com.norconex.commons.norconex-commons-lang;bundle-version="1.15.0"
```

Comment:
Didn't know you were using norconnex for this. I'm using it in JsonOverlay to canonicalize URLs, to maximize likelihood of using shared parse results when two different-looking references actually refer to the same document.

Since I'm already using it, I'll look into it for validations like email as well.